src/types/index.ts

```diff
@@ -0,0 +1,31 @@
+export interface Document {
+    name?: string;
+    original_location?: string;
+}
+
+export type FirestoreDoc = Document & {
+    id: string;
+};
+
+export type Accumulator = {
```

Comment:
Since this type is identical to `StringDict`, could we just get rid of this and use `StringDict` as the type in `getLocationDict`?

Reply:
Make sense! Fixed!